src/net_processing.cpp

```diff
@@ -3231,7 +3234,7 @@ void PeerManagerImpl::ProcessMessage(CNode& pfrom, const std::string& msg_type,
         }
 
         LOCK(cs_main);
-        if (m_chainman.ActiveChainstate().IsInitialBlockDownload() && !pfrom.HasPermission(NetPermissionFlags::Download)) {
+        if (m_chainman.IsAnyChainInIBD() && !pfrom.HasPermission(NetPermissionFlags::Download)) {
```

Comment:
In commit "p2p: don't advertise until we finish all IBDs" (c4d90167f030547373715a44fc020d0da8461565)

There are a few other `ActiveChainstate().IsInitialBlockDownload()` calls in this file that aren't replaced by `m_chainman.IsAnyChainInIBD()` calls. Will these need to be changed later to avoid changing network behavior when an assumeutxo snapshot is loaded? If not, maybe it is worth going through the calls in this PR and adding comments that could shed more light on why message processing is affected by IBD status, or why it only depends on active IBD status not full IBD status.

Reply:
Good point; I've added some commentary to the other uses of IsInitialBlockDownload().